pytransform3d/transform_manager/_temporal_transform_manager.py

```diff
@@ -52,6 +56,79 @@ def check_transforms(self):
         return self
 
 
+class NumpyTimeseriesTransform(TimeVaryingTransform):
+    """Transformation, which does change over time, represented in a pandas
+    dataframe.
+
+    Parameters
+    ----------
+    time_arr: np.ndarray, shape (N,)
```

Comment:
Type names as parts of variable names is like programming in the 80s. :) I'd suggest to call the variables `time` and `pqs`.

Reply:
I adapted, now `time` stands for the time array, and `query_time` for the $`t_q`$

Reply:
Please also update the docstring to match the argument name

Reply:
Adapted